ci/Dockerfile-envoy-alpine

```diff
@@ -2,7 +2,7 @@ FROM frolvlad/alpine-glibc:alpine-3.12_glibc-2.31
 RUN mkdir -p /etc/envoy
```

Comment:
While we are modifying this file, would you be open to bumping the Alpine version to `FROM hfrolvlad/alpine-glibc/alpine-3.13_glibc-2.32` to close out https://github.com/envoyproxy/envoy/issues/14869. /cc @sagarkal. Or would you rather I raise a separate PR @htuch